docs/index.rst

```diff
@@ -1,78 +1,52 @@
-Baseplate
-=========
+Baseplate.py
+============
 
-Baseplate is a framework to build services on and a library of common and
-well-tested code to share. Its goal is to provide all the common things a
-service needs with as few surprises as possible. It can be divided up into
-three distinct categories of components, listed below.
+It's much easier to manage a bunch of services when they all have the same
+shape: the way they're developed, the way they interact with the infrastructure
+they run on, and the way they interact with each other.  Baseplate is reddit's
```

Comment:
I feel so petty in raising this but this period is double spaced whereas the rest of the docs seem to mostly be single-spaced.

Reply:
ha! weird. My personal preference would be for double spaces in the source (since it's monospace) and let smartypants/sphinx correct the typography for the actual renderings. That said, it looks like I've been horribly inconsistent about this but leaning towards single space after period.

```console

$ git grep '[^.]\. \S' | wc -l
547

$ git grep '\.  ' | wc -l
120
```

Fixed this one.